pages/core-unit/[code]/index.tsx

```diff
@@ -15,7 +15,7 @@ export const getServerSideProps: GetServerSideProps = async(context: GetServerSi
   const { query } = context;
   const code = query.code as string;
   const cuAbout = await fetchCoreUnitByCode(code);
-  const contributorCommitment = cuAbout.contributorCommitment;
+  const contributorCommitment = cuAbout && cuAbout.contributorCommitment;
```

Comment:
Please, use optional chaining, it is more readable

`cuAbout?.contributorCommitment;`

Reply:
thanks for the advice, i already made the change